.github/workflows/issue_labeler.yml

```diff
@@ -0,0 +1,38 @@
+# Inspired by: https://github.com/stefanbuck/ristorante
+# See: https://stefanbuck.com/blog/codeless-contributions-with-github-issue-forms
```

Comment:
@jamacku could you remove these links? Those actions are vulnerable to script injections. What's worse they are run with tokens that can be used to write directly to those repositories. In other words they allow anybody on GH to do absolutely anything with those repositories. I don't think the systemd repository should promote that. I haven't looked at that parser closely but pinning it to SHA1 and limiting its permissions should be enough for now to make sure that it isn't going to push commits to the systemd repository.

Reply:
-> https://github.com/systemd/systemd/pull/23814

Reply:
I opened https://github.com/stefanbuck/github-issue-parser/issues/24. Let's see how it goes.